recipe/meta.yaml

```diff
@@ -10,14 +10,15 @@ source:
   md5: 92dd7de2621a6ab6861cc9b689e86c3e
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
   osx_is_app: True
```

Comment:
Does it really make sense to duplicate the entry points if they are already in setup.py by the way?